_posts/2015-06-07-cosmic-neutrinos.md

```diff
@@ -0,0 +1,45 @@
+
+We are all bathed in a constant stream of particles knows as cosmic rays[^1], which are produced mostly outside the Solar System.
+These high-energy particles were initially suspected to have Earthly origins, but Victor Hess demonstrated in 1912 that they arrive on our planet from non-terrestrial sources.
+The enigmatic nature of cosmic rays has captivated scientists for decades.
+
+In 1966, Kenneth Greisen as well as Georgiy Zatsepin and Vadim Kuzmin calculated what has since been known as the [Greisen–Zatsepin–Kuzmin (GZK) limit](https://en.wikipedia.org/wiki/Greisen%E2%80%93Zatsepin%E2%80%93Kuzmin_limit), which is an upper bound for the energies of cosmic rays originating from very distant sources, based on extrapolations from lower energies.
+The GZK effect states that ultrahigh-energy cosmic rays (UHECRs) interact with the photos from the cosmic microwave background (CMB) radiation and lose energy in the process.
+The flux of UHECRs has indeed been shown to be suppressed above the GZK limit.
+However, there may be alternative explanations as to why this is the case; these explanations can be tested by probing the composition, and therefore the sources, of the UHECRs.
+
+One such probe involves searching for ultrahigh-energy cosmic neutrinos.
+Neutrinos are particles belonging to the same family as the electron but are extremely weekly interacting, meaning that they can pass through lightyears of lead without interacting in any way with the dense metal.
+If these neutrinos are produced through the interactions of “primary particles” of cosmic rays with the CMB, they will provide valuable information about the nature of the primary particles, since protons are expected to produce greater numbers of neutrinos through these interactions than heavier primary particles.
+On the other hand, cosmic neutrinos that are the result of the transformation of charged [pions](https://simple.wikipedia.org/wiki/Pion) produced at the cosmic-ray source may be the only cosmic-ray particles capable of travelling distances greater than around 100 megaparsecs or 360 million lightyears.
+Observing and studying these kinds of neutrinos would advance our understanding of astrophysics and help validate or rule out some of the theories that attempt to explain the UHECR energy spectrum and help determine the nature of very distant cosmic-ray sources.
+
+The [Pierre Auger Observatory](https://www.auger.org/) in Argentina has a detection area of around 3000 km<sup>2</sup> and is operated by an international collaboration of more than 500 scientists.
+The Surface Detector (SD) array of the observatory can detect neutrinos that have energies of around and over 1 exaelectronvolt (10<sup>18</sup> eV), which is about [150,000 times the energy of the proton beams in the Large Hadron Collider](http://www.wolframalpha.com/input/?i=1+EeV+vs+6.5+TeV).
```

Comment:
The LHC comparison makes sense to you but not the average person. Spell it out in simpler terms